scripts/deployServices.sh

```diff
@@ -2,21 +2,18 @@
 
 set -e
 
-cd packages/infrastructure/stage-1
+cd packages/charts
```

Comment:
cd charts ?
Siden mappe struktur er endra

Reply:
Kommandoene under kjører er skrevet for å kjøres i mappa charts. Det som var der før navigerte inne i infrastructure og så tilbake til charts. 

Mappe strukturen i dette repoet tenker jeg ikke skal endres. Altså charts skal fortsette å ligge inne i packages

Reply:
Aha, trodde charts skulle flyttes til DevOps. Da er den good - bare å avslå endringen :)